lib/wordpress.js

```diff
@@ -44,7 +44,9 @@ function Client( settings ) {
 		// via settings.host
 		headers: {
 			Host: parsedUrl.host
-		}
+		},
+
+		basic_auth: settings.basic_auth
```

Comment:
the key `basic_auth` is defined in https://github.com/baalexander/node-xmlrpc/blob/master/lib/client.js#L61 so cannot be changed.